File: CodeWars/CodeGarden/13_PrinterErrors.cs
using System;

public class Printer 
{

 static void Main()
    {
        Console.WriteLine("Main Starting...");
        PrinterError("abcxxx");
    }


    public static string PrinterError(String s) 
    {
        int charCount = 0;
        int errorCount = 0;
      
        charCount = s.Length;
      
        for(int i=0; i<=charCount-1; i++)
        {
            var currentLetter = s[i];
        
            if(currentLetter > 'm')
            {
                errorCount++;
            }
        }
        Console.WriteLine(errorCount + "/" + charCount);
        return errorCount + "/" + charCount;
    }
}

File: CodeWars/CodeGarden/13_PrinterErrors.js

function printerError(s) {
    
    let charCount = 0;
    let errorCount = 0;

    charCount = s.length;

    for(let i=0; i<=charCount-1; i++)
    {
        const currentLetter = s[i];

        // js automatically converts for the comparison
        if(currentLetter > 'm')
        {
        // add 1 to errorCount;
        errorCount++;
        }

    }

    // return the number of errors over the length of string
    return `${errorCount}/${charCount}`;

}


const s = "abcxx";
console.log(printerError(s));